src/ViewHelpers/DebugViewHelper.php

```diff
@@ -70,6 +64,83 @@ public function render() {
 			return (is_object($expressionToExamine) ? get_class($expressionToExamine) : gettype($expressionToExamine));
 		}
 
-		return var_export($expressionToExamine, TRUE);
+		$html = $this->arguments['html'];
```

Comment:
It should be possible to avoid double encoding HTML entities - you are of course correct about the escaping concerns, I'll update the patch.